mmv1/templates/terraform/examples/identity_platform_project_default_config.tf.erb

```diff
@@ -18,4 +18,26 @@ resource "google_identity_platform_project_default_config" "<%= ctx[:primary_res
             }
         }
     }
-}
\ No newline at end of file
+
+    blocking_functions {
+        triggers {
+            "beforeSignIn" = {
+                function_uri = "new_uri-before-sign-in"
+            }
+        }
+
+        forward_inbound_credentials {
+            refresh_token = true
+            access_token = true
+            id_token = true
+        }
+    }
+
+    quota {
+        sign_up_quota_config {
+            quota = 1000
+        }
+    }
+
+    authorized_domains = ["test.domain.com", "new..domain.com"]
```

Comment:
Since this will go into the docs, how about let's put the default domains here? This way even if they didn't visit Firebase's docs, they still have some clue to follow.

`authorized_domains = ["localhost", "${projectId}.firebaseapp.com", "${projectId}.web.app"]`